scripts/changelog.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
Again, copied from shx here.  As we round this out, I'll likely release an npm module for this so we don't have to update many scripts.